pkg/server/span_stats_test.go

```diff
@@ -402,15 +402,21 @@ func TestSpanStatsMultiStoreReplicationOff(t *testing.T) {
 func TestSpanStatsFanOutFaultTolerance(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.UnderStressWithIssue(t, 108534)
+
+	// This test simulates a specific set of failures and verifies that
+	// the response collects them appropriately. It is flaky under stress
+	// and race.
+	skip.UnderStress(t)
+	skip.UnderRace(t)
```

Comment:
nit: there is also skip.UnderDuress() which includes `syncutil.DeadlockEnabled`. Not sure if you want to skip under deadlock too

Reply:
I'll just skip the specific scenarios that I've seen it fail under for now.